### DISCUSSION TITLE
Submitting a language pack?

### DISCUSSION THREAD

**Discussion by hsnamr** [AnySoftKeyboard/AnySoftKeyboard#3125] [Show and tell]
Hello All,

I have developed a language pack for Old North Arabian, and I would like to know how can I share it? I tried working with the LanguagePack repositories and following the steps, but ran into many issues with gradle as it appears the LanguagePack repos are not maintained anymore. So in the end I downloaded this repo, and manually added the language module by starting from a copy of another module.

What I would like to know is:
1. Is there a proper process to follow from fork to merge?
2. Can I submit what I have, what would be the preferred method?
3. Are there any limitations on how many language packs I can share?

I was able to test in an emulator and on a hardware, the language pack works. Of course I am abiding by the terms and license of this project. All I did was change a few XML files.

![Screenshot_1639766936](https://user-images.githubusercontent.com/7531006/147773523-c74af586-991c-419c-9e5c-4594e1ee1526.png)

**Comment by menny** [comment]
thank you for the offer. I would love to accept this pack.
We moved all the language packs into this repository under https://github.com/AnySoftKeyboard/AnySoftKeyboard/tree/master/addons/languages

You can read the contribution doc here: https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/addons/CONTRIBUTING.md

**Reply by hsnamr** [reply]
Thank you. I will read the document and follow it, once I figure out the process, I would love to contribute even more language packs.

**Reply by menny** [reply]
Please, do! I will help where I can

**Reply by hsnamr** [reply]
Is it possible to skip adding the dictionaries? These are ancient scripts and I know of no dictionary for them.

Or perhaps I can create a blank dictionary? What do you recommend?
